django/contrib/admin/checks.py

```diff
@@ -22,35 +22,35 @@ def check_admin_app(**kwargs):
 
 class BaseModelAdminChecks(object):
 
-    def check(self, cls, model, **kwargs):
+    def check(self, obj, model, **kwargs):
```

Comment:
Now that all provided objects (`ModelAdmin` instances and `InlineModelAdmin` subclasses) are bound to a model I think we could remove the `model` argument from the methods signature and use `obj.model` instead.